mantidproject/mantid
# 29205 change method and file names in Indirect Data Analysisfor consistency

**Description of work.**

This is the first step in the QENS refactor, for consistency and to make the code base easier to work in I have renamed several classes and files for consistency.

**To test:**

After building the solution open indirect data analysis and test that each tab functions. these changes should not have any effect on the functionality of the tabs.

Fixes #29205

*This does not require release notes* because It is strictly an internal change to the code base with no outward effect.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
